### ISSUE TITLE
Model summary not available

### ISSUE THREAD

**Issue by kabyanil** [tensorflow/models#10160]
I trained my model and saved it in Tensorflow SavedModel format. How can I view the layers of the model in a similar manner to Keras model.summary()? Often times developers want to see what's happening under the hood. So it would be nice to have a feature to check model layers.

**Comment by ymodak** [comment]
Have you tried using GitHub - lutzroeder/netron: Visualizer for neural network, deep learning, and machine learning models 6 to visualize your model.
You may try using browser version that allows you quickly upload your model and it displays op level graph.

**Comment by kabyanil** [comment]
hi, i tried netron. but i wasn't able to get useful info out of it. 

<img width="1269" alt="Screenshot 2021-08-05 at 1 30 02 AM" src="https://user-images.githubusercontent.com/24878109/128246955-11b175e3-07e7-4cc2-afab-a5c641f9d431.png">

I don't understand what these nodes mean. i was looking for a layer structure similar to keras model.summary().

**Comment by ymodak** [comment]
Looks like you are interested in conceptual graph of your model. In that case, TensorBoard provides an option to visualize the conceptual graph in addition to op level graph as displayed by the netron.
You may want to try examining your graph as shown [here](https://www.tensorflow.org/tensorboard/graphs#conceptual_graph).
Thanks!

**Comment by kabyanil** [comment]
<img width="625" alt="Screenshot 2021-08-06 at 10 52 19 PM" src="https://user-images.githubusercontent.com/24878109/128548797-4116d843-7adb-4ad8-94b5-9bd0de7b6f1e.png">

in my case, the option was not available, as you can see from the image. btw, I'm using a script `https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/tools/import_pb_to_tensorboard.py` to load my graph in tensorboard. before running that script, my tensorboard doesn't show the graphs tab. only after running the script, I'm able to load the graph. am I doing anything wrong here?

**Comment by ymodak** [comment]
`import_pb_to_tensorboard` is deprecated. You need to have tensorboard logs to examine graph. 
See https://www.tensorflow.org/tensorboard/graphs#train_the_model_and_log_data

**Comment by kabyanil** [comment]
how can I have tensorboard logs when using TFOD API? Which file should I edit? It's not the same as shown in the example.

**Comment by ymodak** [comment]
You may view the architecture of pre trained model used as feature map extractor in object detection models by using keras.
For instance in SSD model you can view VGG16 model as a feature extractor with
```python
base_model = tf.keras.applications.vgg16.VGG16((input_shape=IMG_SIZE, include_top=False, weights='imagenet')
base_model.summary()
```
You may try other models listed in https://www.tensorflow.org/api_docs/python/tf/keras/applications as well

**Comment by iumyx2612** [comment]
I have this also, I asked [here](https://github.com/tensorflow/tensorboard/issues/5213) on Tensorboard github, you can go check it out for a more detailed visualization of your model
